test/client-API/mailvelope-client-api-test.js

```diff
@@ -0,0 +1,26 @@
+import {expect} from 'test';
+import {LocalStorageStub} from 'utils';
+import mvelo from 'lib/lib-mvelo';
+import * as main from 'controller/main.controller';
+import * as api from 'content-scripts/clientAPI';
+
+describe('Mailvelope Client API', async () => {
+
+  api.init();
```

Comment:
These should probably go somewhere else.
They may only be called once otherwise event handlers will be registered multiple times leading to a single event triggering them multiple times.